conf/generator.yaml

```diff
@@ -145,3 +160,102 @@ modules:
       - artifactid: spring-boot-starter-test
         groupid: org.springframework.boot
         scope: test
+  - name: tracing
+    description: "Distributed tracing with OpenTracing"
+    guide_ref: ""
+    tags:
+      - tracing
+      - monitoring
+    dependencies:
+      - artifactid: opentracing-spring-jaeger-web-starter
+        groupid: io.opentracing.contrib
+      - artifactid: spring-boot-starter
+        groupid: org.springframework.boot
+      - artifactid: spring-boot-starter-test
+        groupid: org.springframework.boot
+        scope: test
+  - name: infinispan
+    description: "Remote Infinispan cache and datagrid"
+    guide_ref: ""
+    tags:
+      - persistence
+      - cache
+    dependencies:
+      - artifactid: infinispan-spring-boot-starter-remote
+        groupid: org.infinispan
+      - artifactid: spring-boot-starter-test
+        groupid: org.springframework.boot
+        scope: test
+  - name: infinispan-embedded
+    description: "Embedded Infinispan cache and datagrid"
+    guide_ref: ""
+    tags:
+      - persistence
+      - cache
+    dependencies:
+      - artifactid: infinispan-spring-boot-starter-embedded
+        groupid: org.infinispan
+      - artifactid: spring-boot-starter-test
+        groupid: org.springframework.boot
+        scope: test
+  - name: artemis
+    description: "Artemis message broker"
+    guide_ref: ""
+    tags:
+      - messaging
+      - jms
+    dependencies:
+      - artifactid: spring-boot-starter-artemis
+        groupid: org.springframework.boot
+      - artifactid: spring-boot-starter-test
+        groupid: org.springframework.boot
+        scope: test
+  - name: activemq
+    description: "ActiveMQ message broker"
+    guide_ref: ""
+    tags:
+      - messaging
+      - jms
+    dependencies:
+      - artifactid: spring-boot-starter-activemq
+        groupid: org.springframework.boot
+      - artifactid: spring-boot-starter-test
+        groupid: org.springframework.boot
+        scope: test
+  - name: amqp
+    description: "AMQP client with JMS API"
```

Comment:
Description should be `AMQP v1.0 client with JMS API`

Reply:
ok